tracing-agent/src/main/java/io/strimzi/tracing/agent/TracingAgent.java

```diff
@@ -22,13 +23,22 @@
      * @param agentArgs The type of tracing which should be initialized
      */
     public static void premain(String agentArgs) {
-        if ("jaeger".equals(agentArgs)) {
+        if (agentArgs != null && agentArgs.contains("jaeger")) {
             String jaegerServiceName = System.getenv("JAEGER_SERVICE_NAME");
```

Comment:
Is the `JAEGER_SERVICE_NAME` used even for OpenTelemetry?

Reply:
I guess this is what they call OTEL_EXPORTER_JAEGER_ENDPOINT ?
So I should map this in the scripts as well ...

Reply:
Actually, the `JAEGER_SERVICE_NAME` is the name under which the traces from this service would be seen in Jaeger. So it is not an endpoint. But not sure if it is relevant for Open Telemetry.

Reply:
OK, I added support for OTEL_SERVICE_NAME ...